keyboards/keychron/q0/config.h

```diff
@@ -21,9 +21,10 @@
 /* turn off effects when suspended */
 #define RGB_DISABLE_WHEN_USB_SUSPENDED
 
-/* EEPROM Driver Configuration */
-#define WEAR_LEVELING_LOGICAL_SIZE 2048
-#define WEAR_LEVELING_BACKING_SIZE (WEAR_LEVELING_LOGICAL_SIZE * 2)
+/* Encoder Configuration*/
+#ifdef ENCODER_ENABLE
+    #define ENCODER_DEFAULT_POS 0x3
+#endif
```

Comment:
```suggestion
#define ENCODER_DEFAULT_POS 0x3
```
Shouldn't do anything if encoder is disabled.

Reply:
Would it be better if I'd just move `#define ENCODER_DEFAULT_POS 0x3` into `q0/rev_0131/config.h` instead?

Reply:
Shouldn't matter, but if it's needed for multiple versions, would be simpler here.

Reply:
Dealt with https://github.com/qmk/qmk_firmware/pull/22068/commits/6060f897b076497dff3fe64b5023ee02e3c2636d.